app/views/admin/items/index.html.haml

```diff
@@ -16,4 +16,4 @@
           %td= link_to item.name
           %td= item.description
           %td= item.status
-          %td= button_to "Edit", edit_admin_item_path(item)
+          %td= button_to "Edit", edit_admin_item_path(item), method: :get
```

Comment:
@tmikeschu Why a `button_to` with `method: :get` instead of a `link_to` here?

https://stackoverflow.com/questions/10374322/haml-link-to-vs-button-to